templates/entry/slim.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Slim entry template.
+ *
+ * @package WPAMPTheme
+ */
+
+?>
+<div id="<?php echo esc_attr( get_the_ID() ); ?>-post" data-sort-time="<?php echo esc_attr( get_the_date( 'U' ) ); ?>" class="col-12 sm-col-6 lg-col-4 p1 travel-results-result" style="float:left;">
+	<div class="relative travel-results-result">
+		<a class="travel-results-result-link" href="<?php the_permalink(); ?>">
+			<amp-img class="rounded bg-silver mb1" width="2" height="1" noloading="" layout="responsive" src="<?php echo esc_url( wp_get_attachment_url( get_post_thumbnail_id() ) ); ?>" ></amp-img>
+		</a>
+	</div>
+	<div class="h2 line-height-2 mb1">
+		<span class="travel-results-result-text"><?php echo get_the_title(); ?></span>
+		<span class="travel-results-result-subtext h3">&bull;</span>
+		<span class="travel-results-result-subtext h3">$&nbsp;</span>
+		<span class="bold"><?php echo esc_html( get_post_meta( get_the_ID(), 'amp_travel_price', true ) ); ?></span>
```

Comment:
What do you think about outputting the 3 lines above only if the `'adventure'` has a price?

Reply:
Nice use of [entry/meta/price.php](https://github.com/xwp/travel/pull/45/commits/6e69f19c8e962a92c3777c67a7ea5c77ad03f8df#diff-a044d91ea4383c8b07d6cb9d97b277fcR1) for this.